app/javascript/admin/controllers/modal_controller.js

```diff
@@ -40,3 +40,6 @@ export default class ModalController extends Controller {
     );
   }
 }
+
+// This is so the controller still loads properly in the Admin section
+export default ModalController;
```

Comment:
In order for this feature to work as expected, we had to move the `export default ModalController;` to the bottom of the file, rather than calling it inline at the top, as shown in the diff.